src/main/java/io/vertx/mqtt/MqttClientOptions.java

```diff
@@ -230,6 +232,17 @@ public boolean isAutoGeneratedClientId() {
     return isAutoGeneratedClientId;
   }
 
+  public int getMaxMessageSize() {
+    return maxMessageSize;
+  }
+
+  public MqttClientOptions setMaxMessageSize(int maxMessageSize) {
+    Arguments.require(maxMessageSize > 0 , "maxMessageSize must be > 0");
+    setReceiveBufferSize(maxMessageSize);
+    this.maxMessageSize = maxMessageSize;
+    return this;
+  }
+
```

Comment:
I know that ReceiveBufferSize and MaxMessageSize are different things but in fact, MQTT client(as well as MQTT server) deal only with MQTT packets